core/src/test/java/org/mule/management/stats/ComponentStatisticsTestCase.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.management.stats;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mule.tck.junit4.AbstractMuleTestCase;
+
+/**
+ * Validates some basic assumptions about the ComponentStatistics class
+ * behavior.
+ */
+public class ComponentStatisticsTestCase extends AbstractMuleTestCase {
+
+    /* This class was added to address MULE-6417 */
+    
+    private String originalStatIntervalTime;
```

Comment:
I recently realised there's a rule for achieving this. For instance, you could do something like:

```
@Rule
public SystemProperty statIntervalTime = new SystemProperty("statIntervalTime", null);
```

to replace the setUp and tearDowm methods.